src/api/templates/index.html

```diff
@@ -12,14 +12,46 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootswatch@5.2.3/dist/cosmo/bootstrap.min.css"
         integrity="sha256-axRDISYf7Hht1KhcMnfDV2nq7hD/8Q9Rxa0YlW/o3NU=" crossorigin="anonymous">
     <link href="/static/styles.css" rel="stylesheet" type="text/css">
+    <style>
+        #messages {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            text-align: center;
+        }
+        #placeholder-wrapper {
+            display: flex;
+            flex-grow: 1;
+            justify-content: center;
+            align-items: center;
+            width: 100%;
+        }
+        #placeholder-message {
+            color: #6c757d;
+        }
+        .header {
+            font-size: 2em;
+            font-weight: bold;
+            margin-bottom: 10px;
+        }
+        .message-content {
+            text-align: left;
+        }
+    </style>
 </head>
 <body>
     <div class="container-fluid h-100 d-flex flex-row">
         <div class="row flex-grow-1 h-100">
             <!-- Chat Section -->
             <div class="col-full d-flex flex-column h-100" id="chat-container"> <!-- Full width initially -->
                 <div id="messages" class="px-4 pb-4 pt-2 flex-grow-1 overflow-y-auto align-items-stretch">
-                    <!-- Message content goes here -->
+                    <div id="placeholder-wrapper">
+                        <div id="placeholder-message" class="text-center text-muted my-3">
+                            <div class="header">Getting Started with Agents Using Azure AI Foundry</div>
+                            Type your message in the text box below. You can start with a simple "Hey" or try something fun like "Tell me a joke."<br>
```

Comment:
We could have this even simpler like

Type your message below. You can start with something friendly like "Hello there," or ask specifically about the Azure Search files, such as "What is Contoso Galaxy Innovations product?"

Reply:
Or

Type your message below. You can start casually with something like "Tell me a joke," or ask specifically about the Azure Search files, such as "What is Contoso Galaxy Innovations product?"